diff --git a/copy_jdk_configs.lua b/copy_jdk_configs.lua
index 801c4d0..426efb3 100755
--- a/copy_jdk_configs.lua
+++ b/copy_jdk_configs.lua
@@ -14,9 +14,17 @@ local caredFiles = {"jre/lib/calendars.properties",
               "jre/lib/deployment.properties",
               "jre/lib/deployment.config",
               "jre/lib/security/US_export_policy.jar",
+              "jre/lib/security/unlimited/US_export_policy.jar",
+              "jre/lib/security/limited/US_export_policy.jar",
+              "jre/lib/security/policy/unlimited/US_export_policy.jar",
+              "jre/lib/security/policy/limited/US_export_policy.jar",
               "jre/lib/security/java.policy",
               "jre/lib/security/java.security",
               "jre/lib/security/local_policy.jar",
+              "jre/lib/security/unlimited/local_policy.jar",
+              "jre/lib/security/limited/local_policy.jar",
+              "jre/lib/security/policy/unlimited/local_policy.jar",
+              "jre/lib/security/policy/limited/local_policy.jar",
               "jre/lib/security/nss.cfg",
               "jre/lib/security/cacerts",
               "jre/lib/security/blacklisted.certs",
